okhttp/src/main/kotlin/okhttp3/ConnectionPool.kt

```diff
@@ -92,4 +125,34 @@ class ConnectionPool internal constructor(
   fun evictAll() {
     delegate.evictAll()
   }
+
+  /**
+   * Sets a policy that applies to [address].
+   * Overwrites any existing policy for that address.
+   */
+  @ExperimentalOkHttpApi
+  fun setPolicy(
+    address: Address,
+    policy: AddressPolicy,
+  ) {
+    delegate.setPolicy(address, policy)
+  }
+
+  /**
+   * A policy for how the pool should treat a specific address.
+   */
+  class AddressPolicy(
+    /**
+     * How many concurrent calls should be possible to make at any time.
+     * The pool will routinely try to pre-emptively open connections to satisfy this minimum.
+     *
+     * This feature must not be used on Android because it may result in network traffic while
+     * the app is backgrounded. On Android, leave this set to 0.
+     */
+    @JvmField val minimumConcurrentCalls: Int = 0,
```

Comment:
I wonder if we should use the word ‘Streams’ or ‘Connections’

Reply:
Maybe Calls is the existing thing we have for streams anyway? That might be fine